sagemaker-inference-recommender/python-sdk-inference-recommender/inference-recommender-with-python-sdk.ipynb

```diff
@@ -0,0 +1,654 @@
+{
```

Comment:
<p>Download data from our S3 Bucket instead to avoid broken links in the future (we uploaded your file to it at s3://sagemaker-sample-files/models/california-housing/model.joblib)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3781/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>